publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/controller/ProducerController.java

```diff
@@ -377,10 +435,10 @@ public JsonObject getAsJsonObject(JsonElement jsonElement) {
      * @param jsonElement AN event which need to check
      * @return boolean type value like it is valid or not
      */
-    private  Boolean isValid(JsonElement jsonElement) {
+    private Boolean isValid(JsonElement jsonElement) {
         try {
             return getAsJsonObject(jsonElement).has(META) && getAsJsonObject(jsonElement).has(DATA) &&
-                    getAsJsonObject(jsonElement).has(LINKS) && !getAsJsonObject(jsonElement).has(STATUS_CODE);
+                    getAsJsonObject(jsonElement).has(LINKS) && !getAsJsonObject(jsonElement).has(JSON_STATUS_CODE);
         } catch (IllegalStateException e) {
             log.error("Error while validating event: ", e.getMessage());
```

Comment:
Event ID should be published whenever a message related to event is produced.

Reply:
Something like ...get(META).get(ID), but I haven't checked...

Reply:
Yeah it can do like this but not sure about the palace you suggest to put these lines, It will benefit anyway or not, WE can discuss once